sdk/src/proto/network.rs

```diff
@@ -1,125 +1,234 @@
 // This file is @generated by prost-build.
+/// The request for a nonce. Should be called before signing a message to get the correct nonce for
+/// the account.
 #[derive(serde::Serialize, serde::Deserialize)]
 #[allow(clippy::derive_partial_eq_without_eq)]
 #[derive(Clone, PartialEq, ::prost::Message)]
 pub struct GetNonceRequest {
+    /// The address to get the nonce for.
     #[prost(bytes = "vec", tag = "1")]
     pub address: ::prost::alloc::vec::Vec<u8>,
 }
+/// The response for a nonce request.
 #[derive(serde::Serialize, serde::Deserialize)]
 #[allow(clippy::derive_partial_eq_without_eq)]
 #[derive(Clone, PartialEq, ::prost::Message)]
 pub struct GetNonceResponse {
+    /// The nonce for the given address. Should be signed along with the rest of the message.
```

Comment:
nit: the rest of what message?

The nonce for the given address. Retrieved when signing new messages

Reply:
"Retrieved when signing new messages" I dont really think the server should be too strict about the behavior of the client. For instance, a client might want to cache the account's nonce and increment on their end to avoid additional RPC calls

Reply:
set it as `Get the nonce for a given account. Used to check current nonce, which must match when signing and sending a message.`.